company/security.md

```diff
@@ -1,53 +1,72 @@
 # Security policy
 
-## General Guidelines
+## General guidelines
 
-* Do not store sensitive information in plain text.
+* Do not store confidential or sensitive information in plain text.
 * Do not store sensitive information with any of the "big 10" cloud providers (MS, Google, Apple, Dropbox, etc.). Store information locally, on our internal infrastructure or with smaller providers (possibly non-US based).
 
 ## Passwords
 
-### Computer Passwords
+### Device passwords
 
-* All computers must have a login password.
-* All computers must be locked with a password when kept running and left somewhere, even if only for a few minutes. 
+* Every computer or device must have a login password or pin to secure access.
+* Use random passwords with more than 16 characters.
+* Do not use the same password more than once.
+* All devices must be locked with a password when kept running and left somewhere, even if only for a few minutes.
 
-### Web Application Passwords
+### Password manager
+
+Use a password manager (1Password, LastPass, KeePass, etc.) to store all other passwords so you do not need to remember them and the passwords can be more complex.
+
+The passwords stored in a password manager should be longer randomly generated passwords with a combination of uppercase and lowercase letters, numbers, and symbols.
+
+* Store passwords for company access to 1Password team account.
```

Comment:
Use of https://chrome.google.com/webstore/detail/1password-x-%E2%80%93-password-ma/aeblfdkhhhdcdjpifhhbdiojplfjncoa is recommended over having copy of passwords or desktop apps.